packages/longhorn-1.1/templates/crd-template/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 1.1.2
```

Comment:
Same question as https://github.com/rancher/charts/pull/1639#discussion_r768911771. Can merge if this is okay.